docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -254,16 +254,18 @@ Specify where to output log information. Valid options are:
 * `DEFAULT_BOTH`
 * `FILE`
 
-Defaults to `DEFAULT_BOTH`. When using `BOTH` or `DEFAULT_BOTH`, system updates such as blockchain events
-are displayed on the console, and errors and other information are logged to a file. The log file
-location can be specified with the [`--log-file`](#log-file) command-line option.
+Defaults to `DEFAULT_BOTH`. When using `BOTH` or `DEFAULT_BOTH`, system updates such as blockchain
+events are displayed on the console, and errors and other information are logged to a file. The
+log file location can be specified with the [`--log-file`](#log-file) command-line option.
```

Comment:
Same comment as above re using active voice: 

Specify the log file location ...